packages/common/src/webrtc/Dialog.ts

```diff
@@ -547,6 +552,7 @@ export default class Dialog {
         logger.error(`${this.id} - Unknown local SDP type:`, data)
         return this.hangup({}, false)
     }
+    msg.targetNodeId = this._targetNodeId
```

Comment:
This will be done by the `_execute`  method if the instance variable `nodeId` is defined:

https://github.com/signalwire/signalwire-node/blob/4be6f34eadb9218958b0a8afedced35544dcd6e3/packages/common/src/webrtc/Dialog.ts#L614-L619

My suggestion is to add a `setter` method on Dialog to set his private `_targetNodeId` from the outside and use 
```javascript
dialog.nodeId = this.node_id
``` 
directly from `VertoHandler.ts` here:

https://github.com/signalwire/signalwire-node/blob/4be6f34eadb9218958b0a8afedced35544dcd6e3/packages/common/src/webrtc/VertoHandler.ts#L59-L63